templates/section_page.hbs

```diff
@@ -41,6 +41,43 @@
       {{/if}}
 
       {{pagination}}
+
+      {{#if section.sections}}
+        <div class="section-tree">
+          {{#each section.sections}}
+          <section class="section">
+            <h3 class="section-tree-title">
+              <a href="{{url}}">{{name}}</a>
+            </h3>
+            {{#if articles}}
+              <ul class="article-list">
+                {{#each articles}}
+                  <li class="article-list-item{{#if promoted}} article-promoted{{/if}}">
+                    {{#if promoted}}
+                      <span data-title="{{t 'promoted'}}" class="icon-star"></span>
+                    {{/if}}
+                    <a href="{{url}}" class="article-list-link">{{title}}</a>
+                    {{#if internal}}
+                      <span class="icon-lock" title="{{t 'internal'}}"></span>
+                    {{/if}}
+                  </li>
+                {{/each}}
+              </ul>
+              {{#if more_articles}}
+                <a href="{{url}}" class="see-all-articles">
+                  {{t 'show_all_articles' count=article_count}}
+                </a>
+              {{/if}}
+            {{else}}
```

Comment:
May sections with no articles contain nested sections? If yes, should this `else` branch be removed like the one in the category page?

Reply:
Yep, you are right! Removed that `else` statement as well :)